e2e/next/src/next.test.ts

```diff
@@ -310,56 +319,24 @@ describe('Next.js Applications', () => {
     });
   }, 300000);
 
-  it('should allow using a custom server implementation in TypeScript', async () => {
+  // TODO(jack): still debugging this
+  fit('should allow using a custom server implementation', async () => {
```

Comment:
Is this `fit` for your debugging?

Reply:
yep, something is failing in CI so just want to see what's going on